src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToArrayTransformer.php

```diff
@@ -99,6 +99,8 @@ public function transform($dateTime)
                 // remove leading zeros
                 $entry = (string) (int) $entry;
             }
+            // unset reference to keep scope clear
+            unset($entry);
```

Comment:
If the $entry var is reused later in the same scope, this ensures that the ref won't be reused by mistake. This is not necessary, but it's still cautious, thus good to me. Removing as much refs as possible would be even better, but I think the remaining cases are justified.